### DISCUSSION TITLE
Accept Donations

### DISCUSSION THREAD

**Discussion by pReya** [StaticJsCMS/static-cms#85] [Ideas]
Is there any way we can contribute financially to this project? GitHub Sponsors? Paypal? Patreon? OpenCollective?

You're really doing important work and I'd like to support it, so that this thing has a long and bright future.

**Comment by KaneFreeman** [comment]
I am working on getting that setup. Hopefully should have it setup soon.

**Comment by KaneFreeman** [comment]
The GitHub sponsor button should hopefully be up in the next week or so. Also we are now on Open Collective: https://opencollective.com/staticjscms

**Comment by KaneFreeman** [comment]
The project can now be sponsored directly through Github and the Sponsor button has been added to the top of the repo.
